portal-frontend/src/app/services/document/document.service.spec.ts

```diff
@@ -23,4 +44,27 @@ describe('DocumentService', () => {
   it('should be created', () => {
     expect(service).toBeTruthy();
   });
+
+  it('should make a get request for getUploadUrl', async () => {
+    mockHttpClient.get.mockReturnValue(of({}));
+
+    const res = await service.getUploadUrl('fileNumber', null);
+    expect(service).toBeTruthy();
```

Comment:
Won't this always be true? Also, doesn't test whether GET request is made.

Reply:
Woops! Yep, looks like I copied that from the default unit test and forgot to change it to expect get. Fixed.